rcl/src/rcl/client.c

```diff
@@ -303,6 +303,8 @@ rcl_take_response_with_info(
   RCL_CHECK_ARGUMENT_FOR_NULL(ros_response, RCL_RET_INVALID_ARGUMENT);
 
   bool taken = false;
+  request_header->source_timestamp = 0;
+  request_header->received_timestamp = 0;
```

Comment:
It would be nice to have in `rmw` a `rmw_get_zero_initialized_service_info` function, though I don't mind strongly.

Reply:
This suggestion can be done in a follow up, for now let's go ahead with this and https://github.com/ros2/rmw_fastrtps/pull/378.

Reply:
> It would be nice to have in rmw a rmw_get_zero_initialized_service_info function, 

Would this still be acceptable now, i.e. after the API freeze? I can easily add it, just wondering whether it would be taken.

Reply:
> Would this still be acceptable now, i.e. after the API freeze? I can easily add it, just wondering whether it would be taken.

In general, new API isn't accepted after the freeze. But in this case, it seems to be trivial enough (for me) and it will avoid future errors. @jacobperron do you think it's ok to add that now?

Reply:
I'd rather not change/add any new API now, unless there is really good reason to do so.